packages/frontend/src/components/ElectionForm/CreateElectionDialog.tsx

```diff
@@ -0,0 +1,351 @@
+            <Box sx={{display: 'flex', flexDirection: 'column', alignItems: 'flex-start', textAlign: 'left'}}>
+                <Typography variant="body1">{title}</Typography>
+                <Typography color='gray' variant="body2">{description.split('. ').map(sentence => <>{`${sentence}.`}<br/></>)}</Typography>
+            </Box>
+            <ArrowForwardIosIcon className="startingOptionArrow" sx={{transition: 'padding-left .2s'}}/>
+        </Box>
+    </Button>
+    <Divider/>
+</>
+
+export default () => {
+    const createElectionContext = useContext(CreateElectionContext);
+
+    const [activeStep, setActiveStep] = useState(0);
+
+    const onClose = () => {
+        setActiveStep(0);
+        createElectionContext.closeDialog();
+    };
+
+    const [errors, setErrors] = useState({title: ''});
+
+    const [election, setElection] = useState(defaultElection);
+
+    // TODO: it would be cool to define a useTranslationExt that did the substitutions for me
+    // TODO: automatic tip insertion would also be cool
+    const { t:_t } = useTranslation();
+    let term = election.settings.term_type === undefined? '' : _t(`terms.${election.settings.term_type}.type`);
+    // wrap the standard t property so that we can insert the term election vs poll
+    const t = key => _t(key)
+        .replace('__CAPITALIZED_TERM__', term) 
+        .replace('__LOWERCASE_TERM__', term.toLowerCase()); 
+
+    const { error, isPending, makeRequest: postElection } = usePostElection()
+
+    const authSession = useAuthSession()
+    const navigate = useNavigate()
+
+    useEffect(() => {
+        let q = createElectionContext.quickPoll;
+        if(!createElectionContext.quickPoll) return;
+
+        // quick poll also specifies a number of other settings but we're not keeping those, we're only keeping the information that the user specified
+        setElection({
+            ...defaultElection,
+            title: q.title,
+            races: [
+                {
+                    ...q.races[0],
+                    title: q.title,
+                    candidates: q.races[0].candidates.filter(candidate => candidate.candidate_name.length > 0)
+                }
+            ]
+        }) 
+    }, [createElectionContext.quickPoll])
+
+    const onAddElection = async (election) => {
+        // calls post election api, throws error if response not ok
+        election.owner_id = authSession.getIdField('sub');
+
+        const newElection = await postElection({
+            Election: election,
+        })
+        if (!newElection)  throw Error("Error submitting election");
+
+        navigate(`/${newElection.election.election_id}/admin`)
+        onClose()
+    }
+
+    return <Dialog
+        open={createElectionContext.open}
+        onClose={onClose}
+        scroll={'paper'}
+        sx={{
+            maxHeight: '90%',
+            '@media(max-width: 500px)': {
```

Comment:
I know mui has better syntax for this, but I felt like media queries would be cleaner syntax here. I let's me keep all the changes in a single block, and I don't have to be worried about accidentally overriding any parent properties from Dialog or Paper